backend/digitales_warten.py

```diff
@@ -52,6 +54,40 @@ def create_queue(place_id):
     db.session.commit()
     return jsonify(id=queue.id, name=queue.name)
 
+@app.route('/places/<place_id>/queues', methods=['GET'])
+def get_queue_state(place_id):
+    person_detail_level = request.args.get('personDetails', None)
+    if 'short' != person_detail_level:
+        abort(404)
+
+    place = Place.query.filter_by(id=place_id).first()
+    if place is None:
+        abort(404)
+    
+    attached_queues = Queue.query.filter_by(place=place).all()
+    logging.warning(attached_queues)
+    if not len(attached_queues):
+        logging.warning("no_queues_found")
+        return json.dumps([])
+    queue_states = []
+    for attached_queue in attached_queues:
+        queue_entries = []
+        waiting_entries = Slot.query.filter_by(queue=attached_queue).all()
+        if not len(waiting_entries):
+            for waiting_entry in waiting_entries:
+                entry = { 'id': waiting_entry.id,
+                           'ticketNumber': waiting_entry.ticket_number
+                        } 
+                queue_entries.append(entry)
+        queue = { 'id': attached_queue.id,
```

Comment:
perhaps add an empty line before this one to make it clear that this should be in a different indentation level

Reply:
Fixed